cli/provider/core_service_linux.go

```diff
@@ -50,6 +53,9 @@ func Get(ctx context.Context, cmd string, cfg *config.Config, logger *zap.Logger
 	if cmd == "test" || cmd == "normalize" || cmd == "templatize" {
 		return replaySvc, nil
 	}
```

Comment:
There are so many if blocks here, can you put them into switch case? switch case on cmd?